### ISSUE TITLE
EXCEPTION when Presence message has an empty "from" field

### ISSUE THREAD

**Issue by DocThreeC** [horazont/aioxmpp#241]
By mistake we have tried to subscribe to ourselves and PROSODY returns a PresenceType.ERROR message with an empty from field. This raises the following exception:

Message Received:
```
STANZA:  <presence from=None to='alex@domain/a9da2416-e5a1-4c4c-bc2c-ac6d4db5c8d2' id='xAyYYF8ga8wajUf2Sw4Cs' type=<PresenceType.ERROR: 'error'>>
ERROR:aioxmpp.callbacks:listener attached to signal raised

Traceback (most recent call last):
  File "/Users/carlos/anaconda3/envs/spade/lib/python3.6/site-packages/aioxmpp/callbacks.py", line 536, in fire
    keep = wrapper(args, kwargs)
  File "/Users/carlos/anaconda3/envs/spade/lib/python3.6/site-packages/aioxmpp/callbacks.py", line 470, in _strong_wrapper
    return not f(*args, **kwargs)
  File "/Users/carlos/anaconda3/envs/spade/lib/python3.6/site-packages/aioxmpp/dispatcher.py", line 319, in _feed
    super()._feed(stanza)
  File "/Users/carlos/anaconda3/envs/spade/lib/python3.6/site-packages/aioxmpp/dispatcher.py", line 135, in _feed
    cb(stanza)
  File "/Users/carlos/anaconda3/envs/spade/lib/python3.6/site-packages/aioxmpp/presence/service.py", line 193, in handle_presence
    bare = st.from_.bare()
AttributeError: 'NoneType' object has no attribute 'bare'
```

**Comment by horazont** [comment]
Thanks for reporting this. Obviously a bug. Adding to the 0.11 milestone for now, we might backport a fix to 0.10.

Does this affect your usage in any way?

**Comment by javipalanca** [comment]
Hi, this is not affecting @DocThreeC 's usage. Don't worry ;)
We'll wait for the 0.11 release!
